### PR TITLE
Add a npm command for quicker development

### PR DESCRIPTION
**Pull request by jdlrobson**
webpack watch watches changes to files meaning it's quicker to
make modifications and test the JavaScript

### Description

Closes #

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "repository": "github:internetarchive:openlibrary",
   "license": "AGPL-3.0",
   "scripts": {
+    "start": " webpack -w --mode=development",
```
Comment (cdrini) [inline package.json]: Rename to `watch`; `npm start` is usually used to start a server (not necessarily with watch); `watch` is more common for live compiling of JS/CSS resources.

**Comment by jdlrobson** [comment]
Frontend guide is on the wiki - https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide - I'll update it when this lands.

**Comment by cdrini** [comment]
Sweet! Testing this now :)

**Comment by cdrini** [comment]
I'm having trouble getting this working; if I make edits it does nothing. I tried editing `ol.js` and `header-bar.less` and then waited for a few minutes. Ran `npm update` (just incase) but that didn't seem to work either.

```
$ docker-compose exec web npm run-script watch

> openlibrary@1.0.0 watch /openlibrary
>  webpack -w --mode=development


webpack is watching the files…

Built at: 2019-08-15 16:58:54
```

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "repository": "github:internetarchive:openlibrary",
   "license": "AGPL-3.0",
   "scripts": {
+    "watch": "webpack -w --mode=development",
```
Comment (cdrini) [inline package.json]: Let's enable progress; that makes it much easier to see what's  happening. Also write out the full variable name for clarity.

```suggestion
    "watch": "webpack --watch --progress --mode=development",
```

**Pull request merged**